src/routes/events.geofenced-checkin.test.tsx

```
@@ -0,0 +1,197 @@
+      hearing_loop: false,
+      low_sensory_zone: true,
+    },
+    latitude: 40.7128,
+    longitude: -74.0060,
+    geofence_radius_meters: 150,
+  },
+};
+
+const mockSupabase = {
+  auth: {
+    getSession: vi.fn().mockResolvedValue({ data: { session: { access_token: "mock-token" } }, error: null }),
```

ESLint

[error] Replace `.fn()` with `⏎······.fn()⏎······`

(prettier/prettier)